packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -113,6 +115,11 @@ const confirmRemoval = createAction(REMOVAL_CONFIRMED);
 const remove = createAction(REMOVE, contextPath => contextPath);
 
 /**
+ * Remove all nodes from the store
+ */
+const replace = createAction(REPLACE, ({siteNode, documentNode, nodes}) => ({siteNode, documentNode, nodes}));
```

Comment:
I'd like to rename the variables a little, like: "siteNodeContextPath, documentNodeContextPath, nodes"; or at least document it here.